reconcile/queries.py

```diff
@@ -2434,3 +2434,53 @@ def get_dyn_traffic_directors():
 def get_status_pages():
     gqlapi = gql.get_api()
     return gqlapi.query(STATUS_PAGE_QUERY)['status_pages']
+
+
+CLOSED_BOX_MONITORING_PROBES_QUERY = """
+{
+  apps: apps_v1 {
```

Comment:
why are you querying apps and not the endpoint resources directly?

Reply:
the endpoints are not dedicated "objects" but embedded into an app. i'm not aware on how to query those without starting from apps_v1

but what i can do is remove the fields from app because i don't need them

Reply:
oh, sorry. i misunderstood. please disregard!